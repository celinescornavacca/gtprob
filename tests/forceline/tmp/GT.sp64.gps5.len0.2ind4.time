161.83user 0.76system 2:42.89elapsed 99%CPU (0avgtext+0avgdata 354544maxresident)k
16inputs+8outputs (0major+326865minor)pagefaults 0swaps
