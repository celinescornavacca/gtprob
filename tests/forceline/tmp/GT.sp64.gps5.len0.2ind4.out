-611.223084051
